tests/baselines/reference/jsdocDisallowedInTypescript.errors.txt

```diff
@@ -1,75 +1,75 @@
+tests/cases/conformance/jsdoc/jsdocDisallowedInTypescript.ts(18,17): error TS8020: Expected '}' or ';'.
+tests/cases/conformance/jsdoc/jsdocDisallowedInTypescript.ts(20,16): error TS8020: Expected '}' or ';'.
+tests/cases/conformance/jsdoc/jsdocDisallowedInTypescript.ts(21,16): error TS8020: Expected '}' or ';'.
+tests/cases/conformance/jsdoc/jsdocDisallowedInTypescript.ts(22,17): error TS8020: Expected '}' or ';'.
 
 
 ==== tests/cases/conformance/jsdoc/jsdocDisallowedInTypescript.ts (16 errors) ====
     // grammar error from checker
     var ara: Array.<number> = [1,2,3];
                   ~
-!!! error TS8020: JSDoc types can only be used inside documentation comments.
+!!! error TS8020: Expected '}' or ';'.
     
     function f(x: ?number, y: Array.<number>) {
                   ~~~~~~~
-!!! error TS8020: JSDoc types can only be used inside documentation comments.
+!!! error TS8020: Expected '}' or ';'.
                                    ~
-!!! error TS8020: JSDoc types can only be used inside documentation comments.
+!!! error TS8020: Expected '}' or ';'.
         return x ? x + y[1] : y[0];
     }
     function hof(ctor: function(new: number, string)) {
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS8020: JSDoc types can only be used inside documentation comments.
+!!! error TS8020: Expected '}' or ';'.
         return new ctor('hi');
     }
     function hof2(f: function(this: number, string): string) {
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS8020: JSDoc types can only be used inside documentation comments.
+!!! error TS8020: Expected '}' or ';'.
         return f(12, 'hullo');
                ~~~~~~~~~~~~~~
 !!! error TS2554: Expected 1 arguments, but got 2.
     }
     var whatevs: * = 1001;
                  ~
-!!! error TS8020: JSDoc types can only be used inside documentation comments.
+!!! error TS8020: Expected '}' or ';'.
     var ques: ? = 'what';
               ~
-!!! error TS8020: JSDoc types can only be used inside documentation comments.
+!!! error TS8020: Expected '}' or ';'.
     var g: function(number, number): number = (n,m) => n + m;
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS8020: JSDoc types can only be used inside documentation comments.
+!!! error TS8020: Expected '}' or ';'.
     var most: !string = 'definite';
               ~~~~~~~
-!!! error TS8020: JSDoc types can only be used inside documentation comments.
+!!! error TS8020: Expected '}' or ';'.
     var postfixdef: number! = 101;
                     ~~~~~~~
-!!! error TS8020: JSDoc types can only be used inside documentation comments.
+!!! error TS8020: Expected '}' or ';'.
     var postfixopt: number? = undefined;
         ~~~~~~~~~~
 !!! error TS2322: Type 'undefined' is not assignable to type 'number | null'.
                     ~~~~~~~
-!!! error TS8020: JSDoc types can only be used inside documentation comments.
+!!! error TS8020: Expected '}' or ';'.

```

Comment:
This is much worse. The linked issue is about adding better logic to the parser to issue better error messages, not just changing the text of a particular diagnostic.